client/js/widgets/button.js

```diff
@@ -56,9 +56,13 @@ export class Button extends Widget {
 
     const variables = {
       playerName,
-      playerColor
+      playerColor,
+      thisID : this.p('id')
     };
-    const collections = {};
+
+    const collections = {
+        thisButton : [this]
```

Comment:
Please keep indentation consistent. :)

Reply:
I couldn't even see the issue at first :) but resolved now.